using Xeptions;

namespace $rootnamespace$.$models$.$safeItemNamePlural$.Exceptions
{
    public class $safeItemNameSingular$ValidationException : Xeption
    {
        public $safeItemNameSingular$ValidationException(Xeption innerException)
            : base(message: "$safeItemNameSingular$ validation error occurred, please try again.",
                  innerException)
        { }
    }
}